src/plugins/wb-postback/wb-postback.js

```diff
@@ -41,42 +41,45 @@ var $document = wb.doc,
 
 			// Set "clicked" attribute on element that initiated the form submit
 			$buttons.on( "click", function() {
-				$buttons.removeAttr( attrClick );
-				$( this ).attr( attrClick, "" );
+				$buttons.removeAttr( attrEngaged );
+				$( this ).attr( attrEngaged, "" );
 			} );
 
 			elm.addEventListener( "submit", function( e ) {
 
 				// Prevent regular form submit
 				e.preventDefault();
 
-				var data = $elm.serializeArray(),
-					$btn = $( "[type=submit][" + attrClick + "]", $elm ),
-					$selectorSuccess = $( selectorSuccess ),
-					$selectorFailure = $( selectorFailure );
+				if ( !$( this ).attr( attrEngaged ) ) {
+					var data = $elm.serializeArray(),
+						$btn = $( "[type=submit][" + attrEngaged + "]", $elm ),
+						$selectorSuccess = $( selectorSuccess ),
+						$selectorFailure = $( selectorFailure );
 
-				if ( $btn ) {
-					data.push( { name: $btn.attr( "name" ), value: $btn.val() } );
-				}
-
-				$.ajax( {
-					type: this.method,
-					url: this.action,
-					data: $.param( data )
-				} )
-				.done( function() {
-					$selectorFailure.addClass( classToggle );
-					$selectorSuccess.removeClass( classToggle );
-				} )
-				.fail( function() {
-					$selectorSuccess.addClass( classToggle );
-					$selectorFailure.removeClass( classToggle );
-				} )
-				.always( function() {
-					if ( selectorContent ) {
-						$( selectorContent ).addClass( classToggle );
+					if ( $btn ) {
+						data.push( { name: $btn.attr( "name" ), value: $btn.val() } );
 					}
-				} );
+					$( this ).attr( attrEngaged, true );
+
+					$.ajax( {
+						type: this.method,
+						url: this.action,
+						data: $.param( data )
+					} )
+					.done( function() {
+						$selectorFailure.addClass( classToggle );
+						$selectorSuccess.removeClass( classToggle );
+					} )
+					.fail( function() {
+						$selectorSuccess.addClass( classToggle );
+						$selectorFailure.removeClass( classToggle );
+					} )
+					.always( function() {
+						if ( selectorContent ) {
```

Comment:
Remove the double click prevention flag. Or if we only want to be click once, that option should be turn on by the author via the button or the form.